api/external/envoy/api/v2/core/base.proto

```diff
@@ -1,27 +1,25 @@
 syntax = "proto3";
 
-package envoy.api.v2.core;
+package solo.io.envoy.api.v2.core;
```

Comment:
why add the `solo.io.` prefix everywhere?

Reply:
This is an unfortunate result of the newer versions of `golang/protobuf`. Essentially if a type is registered twice, it logs a very verbose warning, and will start to panic in an upcoming release. A type is registered by it's `package` + `TypeName`. So these types were generated by us and envoy with the same name, causing them to be "registered twice". In order to solve this I prepended a `solo.io.` to all of our copied/owned envoy protos.